examples/helpers/cmake/gen_android_example.cmake.in

```diff
@@ -55,6 +55,12 @@ function(gen_android_example_application
         set(EDGE_SDK_EXAMPLE_NETWORK_PROTOCOL_JAR "-DEDGE_SDK_NETWORK_PROTOCOL_JAR=${EDGE_SDK_NETWORK_PROTOCOL_PATH}/EdgeNetworkProtocol.jar")
     endif()
 
+    set(BOOST_ROOT "full path to the boost library folder in your env")
+    # Try to find the path to the boost
+    if (DEFINED olp-cpp-sdk-core_BINARY_DIR)
```

Comment:
```suggestion
    if (DEFINED BOOST_ROOT)
```
We already define the `BOOST_ROOT` variable when downloading Boost by ourselves - see corresponding file:
https://github.com/heremaps/here-olp-edge-sdk-cpp/blob/master/external/CMakeLists.txt#L50